packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -70,13 +71,17 @@ function aggregateTranslationMessages(options, inputFileSystem) {
   const languageMessages = {};
   supportedLocales.forEach((language) => { languageMessages[language] = {}; });
 
-  // Read translations at the baseDirectory
-  readTranslations(languageMessages, options.baseDirectory, inputFileSystem);
-  // Set translationsDirectory to root node_modules if not specified
-  const translationsDirectory = options.translationsDirectory || ['node_modules'];
-  // Search and aggregate translation messages under the custom directories
-  translationsDirectory.forEach(dir =>
-    aggregateDirectory(languageMessages, path.resolve(options.baseDirectory, dir), inputFileSystem),
+  // Read translations directly from specific paths
+  if (options.translationsDirectories) {
+    options.translationsDirectories.forEach(dir =>
+      readTranslations(languageMessages, path.resolve(options.baseDirectory, dir), inputFileSystem),
+    );
+  }
+  // Set translationsDirectoryRouters to node_modules if not specified
+  const translationsDirectoryRouters = options.translationsDirectoryRouters || ['node_modules'];
```

Comment:
So I am wondering after reading the documentation above if it would make sense to always concat 'node_modules' given this is the purpose of the plugin and it removes the responsibility of the consumer to specify it themselves. IMO consumer would expect this aggregation by default.

```
const translationsDirectoryRouters = options.translationsDirectoryRouters ?options.translationsDirectoryRouters.concat(['node_modules']) || ['node_modules'];
```